.github/workflows/tests.yml

```diff
@@ -98,3 +98,177 @@ jobs:
     - name: Test with pytest
       run: |
         pytest -v ./tests/sealapi
+  
+  TenSEAL_WIN:
+
+    runs-on: windows-latest
```

Comment:
I think there a lot of duplicate code here, you can you something global like

```    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [windows-latest, macos-latest, ubuntu-18.04]
```

And keep a single TenSEAL

You can see a similar example here https://github.com/OpenMined/PSI/blob/master/.github/workflows/tests.yml

Reply:
ok I will make changes